test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -292,7 +292,10 @@ def test_framework_and_neuron_sdk_version(neuron):
         tested_framework = tested_framework[len("huggingface_"):]
 
     if tested_framework == "pytorch":
-        tested_framework = "torch_neuron"
+        if "pytorch-training-neuron" in image:
```

Comment:
is this compatible with previous versions of PT eg. PT1.11?

Reply:
Yes